python/mxnet/module/bucketing_module.py

```diff
@@ -335,12 +339,16 @@ def bind(self, data_shapes, label_shapes=None, for_training=True,
         self._grad_req = grad_req
 
         symbol, data_names, label_names = self._call_sym_gen(self._default_bucket_key)
-        module = Module(symbol, data_names, label_names, logger=self.logger,
-                        context=self._context, work_load_list=self._work_load_list,
-                        fixed_param_names=self._fixed_param_names,
-                        state_names=self._state_names,
-                        group2ctxs=self._group2ctxs,
-                        compression_params=self._compression_params)
+        module = None
+        if not self._default_bucket_key in self._buckets:
```

Comment:
do you have to check that _default_bucket_key is set? If it's none it shouldn't be in the dictionary I assume.